lib/ddtrace/contrib/sidekiq/ext.rb

```diff
@@ -15,6 +16,7 @@ module Ext
         TAG_JOB_QUEUE = 'sidekiq.job.queue'.freeze
         TAG_JOB_RETRY = 'sidekiq.job.retry'.freeze
         TAG_JOB_WRAPPER = 'sidekiq.job.wrapper'.freeze
+        TAG_JOB_ARGS = 'sidekiq.job.arguments'.freeze
```

Comment:
Missed a spot here, and in the corresponding tests.

Reply:
I wanted to leave that as `arguments` as it reads nice in the web UI. I can change to `args` if you feel strongly about it.

Reply:
In this case I think we should, because it makes it consistent with our Rake integration which also tags arguments as `args`.